index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Lynn Jansheski Personal Portfolio</title>
+  <link rel="stylesheet" href="/stylesheets/styles.css">
+  <link rel="stylesheet" href="/stylesheets/fonts.css">
+</head>
+
+  <body class="home">
+
+    <main class="page_grid">
```

Comment:
This main tag here is wrapping all of the content - I recommend moving the class from this tag into the body tag, and replacing the section tag below with main:
```suggestion
  <body class="home page_grid">
```